File: multiple-languages/dotnet/AlibabaCloud.SDK.ROS.CDK.Vpc/AlibabaCloud/SDK/ROS/CDK/Vpc/DhcpOptionsSetAttachmentProps.cs
using Amazon.JSII.Runtime.Deputy;

#pragma warning disable CS0672,CS0809,CS1591

namespace AlibabaCloud.SDK.ROS.CDK.Vpc
{
    #pragma warning disable CS8618

    /// <summary>Properties for defining a `ALIYUN::VPC::DhcpOptionsSetAttachment`.</summary>
    [JsiiByValue(fqn: "@alicloud/ros-cdk-vpc.DhcpOptionsSetAttachmentProps")]
    public class DhcpOptionsSetAttachmentProps : AlibabaCloud.SDK.ROS.CDK.Vpc.IDhcpOptionsSetAttachmentProps
    {
        /// <summary>Property dhcpOptionsSetId: The ID of the DHCP options set.</summary>
        [JsiiProperty(name: "dhcpOptionsSetId", typeJson: "{\"union\":{\"types\":[{\"primitive\":\"string\"},{\"fqn\":\"@alicloud/ros-cdk-core.IResolvable\"}]}}")]
        public object DhcpOptionsSetId
        {
            get;
            set;
        }

        /// <summary>Property vpcId: The ID of the VPC network that is to be associated with the DHCP options set.</summary>
        [JsiiProperty(name: "vpcId", typeJson: "{\"union\":{\"types\":[{\"primitive\":\"string\"},{\"fqn\":\"@alicloud/ros-cdk-core.IResolvable\"}]}}")]
        public object VpcId
        {
            get;
            set;
        }
    }
}

File: multiple-languages/go/alicloudroscdkvpc/DhcpOptionsSetAttachmentProps.go
package alicloudroscdkvpc


// Properties for defining a `DhcpOptionsSetAttachment`.
//
// See https://www.alibabacloud.com/help/ros/developer-reference/aliyun-vpc-dhcpoptionssetattachment
type DhcpOptionsSetAttachmentProps struct {
	// Property dhcpOptionsSetId: The ID of the DHCP options set.
	DhcpOptionsSetId interface{} `field:"required" json:"dhcpOptionsSetId" yaml:"dhcpOptionsSetId"`
	// Property vpcId: The ID of the VPC network that is to be associated with the DHCP options set.
	VpcId interface{} `field:"required" json:"vpcId" yaml:"vpcId"`
}

